src/pages/items/Items.jsx

```diff
@@ -0,0 +1,18 @@
+import './Items.css';
+import BestItems from '../../components/items/BestItems/BestItems';
+import SailItems from '../../components/items/SailItems/SailItems';
+
+function Items() {
+  return (
+    <>
+      <div className="container-items">
+        {/* 베스트 상품 */}
+        <BestItems></BestItems>
+        {/* 판매 중인 상품 */}
+        <SailItems></SailItems>
```

Comment:
자식 컴포넌트가 없는 경우에는 `self closing component`를 사용해주시면 됩니다.

```suggestion
        <BestItems />
        {/* 판매 중인 상품 */}
        <SailItems />
```